html/off_web_html/html

```diff
@@ -0,0 +1 @@
+/path/to/openfoodfacts-web/html
```

Comment:
I think this was added to the PR by mistake as it seems unrelated, can you remove it?

Reply:
Yes @stephanegigandet, I will remove it right away